pages/1.10/deploying-services/marathon-on-marathon/advanced/index.md

```diff
@@ -1,118 +1,138 @@
+-  A private Docker registry that each private DC/OS agent can access over the network. You can follow [these](/1.10/deploying-services/private-docker-registry/) instructions for how to set up in Marathon, or use another option such as [DockerHub](https://hub.docker.com/), [Amazon EC2 Container Registry](https://aws.amazon.com/ecr/), and [Quay](https://quay.io/)).
+-  Custom non-native Marathon (version 0.11.0 or newer) image [deployed in your private docker registry](/1.10/deploying-services/private-docker-registry#tarball-instructions). Contact your sales representative or <sales@mesosphere.io> for obtain the enterprise marathon image file.
 -  You must be logged in as a superuser.
 -  SSH access to the cluster.
 
-# Step 1 - Load and Push the Custom Non-Native Marathon Image
-In this step, the custom non-native Marathon instance is pushed to the private Docker registry.
+<a name="variables-in-example"></a>
+**Variables in this example**
 
-1. Load the tarball into Docker with the custom non-native Marathon file (`marathon-dcos-ee.<version>.tar`) specified. 
+Throughout this page, you will be instructed to invoke commands or take actions that differ according to your set-up. For the parts that differ, we are using the `${VARIABLE}` notation, that should be replaced with the appropriate value that fits in your case.
 
-   ```bash
-   docker load -i marathon-dcos-ee.<version>.tar
-   ```
-     
-    **Tip:** You can view the Marathon image with this command.
+The following table contains all the variables used in this page:
 
-    ```
-    docker images
-    ```
-    
-    You should see output similar to this:
-    
-    ```bash
-    REPOSITORY                    TAG                 IMAGE ID            CREATED             SIZE
-    mesosphere/marathon-dcos-ee   1.4.0-RC4_1.9.4     d1ffa68a50c0        3 months ago        926.4 MB
-    ```
-     
-1. Rename the file to match the repository that you are using in your private Docker registry, where:
-   - `<mesosphere-tag>` is the tag of the image from Mesosphere. Typically, this will match the version number in the filename. 
-   - `<your-repo>` is the name of the private repository that you want to store the image in.
-   - `<your-tag>` is the tag for the image. It is recommended that you use the same tag as the Mesosphere image.
+| Variable | Description |
+|--------------------------|--------------------------------------------|
+| `${MY_USER_ID}` | The username that you used to login to your cluster. That's the username that you were prompted during `dcos cluster setup`. |
+| `${AGENT_ID}` | The ID of an agent in your DC/OS cluster. You can obtain an agent id using the `dcos node` command. |
```

Comment:
`agent id -> agent ID`

Reply:
This is right. Also, English is awful. ID is not an acronym.

Reply:
✅